auth4genai/snippets/get-started/react-spa-vercel-ai/call-others-api.mdx

```diff
@@ -0,0 +1,635 @@
+### Prerequisites
```

Comment:
can you do prerequisites like how its done on other quickstart, for example refer the vercel AI + Next.js ones